### PR TITLE
Modify test cmfgen

### PR DESCRIPTION
**Pull request by ymatsumu**
Add test for cmfgen

**Description**

**Motivation and context**

**How has this been tested?**
- [ ] Testing pipeline.
- [ ] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: carsus/io/tests/test_cmfgen.py [hunk 1]
```diff
@@ -169,15 +169,27 @@ def test_get_seaton_phixs_table(threshold_energy_ryd, fit_coeff_list):
 
 
 @pytest.mark.array_compare
-@pytest.mark.parametrize("hyd_gaunt_energy_grid_ryd", [{1: list(range(1, 3))}])
+@pytest.mark.parametrize("hyd_gaunt_energy_grid_ryd", [{1: list(range(1, 4))}])
 @pytest.mark.parametrize("hyd_gaunt_factor", [{1: list(range(3, 6))}])
 @pytest.mark.parametrize("threshold_energy_ryd", [0.5])
 @pytest.mark.parametrize("n", [1])
+@pytest.mark.parametrize("hyd_n_phixs_stop2start_energy_ratio", [25])
+@pytest.mark.parametrize("hyd_n_phixs_num_points", [200])
```
Comment (chvogl) [inline carsus/io/tests/test_cmfgen.py:177]: For testing, I would set this to a smaller number of points to keep the generated file small. 20 points should be enough?

**Review by chvogl — APPROVED** [review]
Looks good. I would suggest choosing a smaller number of points to keep the generated file small.

File: carsus/io/tests/test_cmfgen.py (hunk 1 above)
Comment (ymatsumu) [reply carsus/io/tests/test_cmfgen.py:177]: I see, thank you for the comments! Should I regenerate a file and push it to carsus-refdata again then?

Comment (chvogl) [reply carsus/io/tests/test_cmfgen.py:177]: That would be great!

**Comment by chvogl** [comment]
Addressed by #2.

**Pull request closed without merging**